### PR TITLE
[permission_handler] Update permission_handler to 8.3.0

### PR DESCRIPTION
**Pull request by swift-kim**
Only Android- and iOS-specific permissions have been added to [permission_handler_platform_interface](https://pub.dev/packages/permission_handler_platform_interface/changelog) (between 3.1.1 and 3.7.0).

### CODE REVIEW

File: packages/permission_handler/tizen/src/permission_handler_tizen_plugin.cc [hunk 1]
```diff
@@ -10,16 +10,13 @@
 #include "service_manager.h"
 
 namespace {
-constexpr char kPermissionHandlerChannelName[] =
```
Comment (bbrto21) [inline packages/permission_handler/tizen/src/permission_handler_tizen_plugin.cc:13]: Do you have a specific reason to remove it?

**Review by bbrto21 — APPROVED** [review]

File: packages/permission_handler/tizen/src/permission_handler_tizen_plugin.cc (hunk 1 above)
Comment (swift-kim) [reply packages/permission_handler/tizen/src/permission_handler_tizen_plugin.cc:13]: Most of the plugins in this repo use string literals as channel names without declaring them as constants.

File: packages/permission_handler/example/tizen/tizen-manifest.xml [hunk 2]
```diff
@@ -1,11 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest package="org.tizen.permission_handler_tizen_example" version="1.0.0" api-version="4.0" xmlns="http://tizen.org/ns/packages">
-    <profile name="common"/>
-    <ui-application appid="org.tizen.permission_handler_tizen_example" exec="Runner.dll" type="dotnet" multiple="false" taskmanage="true" nodisplay="false" api-version="4" launch_mode="single">
+    <profile name="wearable"/>
```
Comment (bwikbs) [inline packages/permission_handler/example/tizen/tizen-manifest.xml:3]: Is this intended?

File: packages/permission_handler/tizen/src/permission_handler_tizen_plugin.cc (hunk 1 above)
Comment (bwikbs) [reply packages/permission_handler/tizen/src/permission_handler_tizen_plugin.cc:13]: I like this way!

**Review by bwikbs — APPROVED** [review]
Thanks :+1:

File: packages/permission_handler/example/tizen/tizen-manifest.xml (hunk 2 above)
Comment (swift-kim) [reply packages/permission_handler/example/tizen/tizen-manifest.xml:3]: Yes :) https://github.com/flutter-tizen/plugins/pull/281#discussion_r760807695

Comment (bwikbs) [reply packages/permission_handler/example/tizen/tizen-manifest.xml:3]: OMG! Everyone thinks the same way! :)

Comment (swift-kim) [reply packages/permission_handler/example/tizen/tizen-manifest.xml:3]: FYI, there's also a written guideline for this. :)

https://github.com/flutter-tizen/flutter-tizen/wiki/New-plugin-submission-guideline#tizen-manifestxml

**Pull request merged**